### PR TITLE
Implement customizable toggle windowed mode shortcut

### PR DESCRIPTION
**Pull request by PurpleMadness**
Closes issue #3421.

### CODE REVIEW

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -4111,6 +4111,7 @@ STR_5802    :{SMALLFONT}{BLACK}Stops guests from littering and vomiting
 STR_5803    :{SMALLFONT}{BLACK}Rotate selected map element
 STR_5804    :Mute sound
 STR_5805    :{SMALLFONT}{BLACK}If checked, your server will be added to the{NEWLINE}public server list so everyone can find it
+STR_5806	:Toggle windowed mode
```
Comment (janisozaur) [inline data/language/en-GB.txt]: this should be aligned

**Comment by PurpleMadness** [comment]
Not sure why the build fails.

**Comment by Gymnasiast** [comment]
AppVeyor choking on NSIS again. That's not caused by your code.

**Comment by IntelOrca** [comment]
This needs rebasing.

**Comment by PurpleMadness** [comment]
That should do it.

**Comment by IntelOrca** [comment]
You should learn to rebase in the future.

**Comment by PurpleMadness** [comment]
What do you mean? Sorry, I'm still pretty new to the gitflow workflow :)

**Comment by janisozaur** [comment]
Rebasing is not specific to gitflow.

When you rebase, it produces cleaner, more manageable and shorter history. It's also much easier to do bisection on.

**Comment by IntelOrca** [comment]
https://help.github.com/articles/about-git-rebase/
https://help.github.com/articles/using-git-rebase/

**Pull request merged**